README.md

```diff
@@ -1,25 +1,89 @@
-# StayFree Destop - MacOS, Windows, and Linux
+# StayFree Desktop - MacOS, Windows, and Linux
 
 This repo holds the release packages for the MacOS, Windows, and Linux versions of StayFree.
 
 These desktop platforms are under heavy development right now and contain a minimal feature set. They will be expanded with the full suite of StayFree features (that our users love!) as we continue to develop them.
 
+## Overview
+
 For the initial release:
-* These apps are just tracking application usage on your computer.
-  * So if you used "Google Chrome" for 2 hours and "Visual Studio Code" for another hour, that is what would show up within the app.
-  * At this time, they do not track website usage within your browser.
-  * Please rely on our browser extensions to track website usage within your browsers: https://stayfreeapps.com
+* **Application Tracking:** The apps currently track usage of applications on your computer. For example, if you used "Google Chrome" for 2 hours and "Visual Studio Code" for another hour, this is what will be recorded in the app.
+* **Website Tracking:** As of now, website tracking within your browser is not supported. For tracking website usage, please use our browser extensions available [here](https://stayfreeapps.com).
  
-Notable features that are missing:
-* App blocking functionality has not been implemented yet. This will come in the next release.
-* The desktop app doesn't yet connect to our cross platform system (so it won't display usage from your phone/browser extension). This will be added in the near future.
-* App icons do not exist yet. These will be implemented as we get a clearer picture around the usage of this version of StayFree.
-* Advanced blocking features like Focus Mode will be implemented in the uncoming releases.
-* UI functionality for more in-depth stats, comparisons, and usability will also be prioritized for the upcoming releases.
-* Over time, the functionality of this app will closely resemble that of our browser extensions.
+
+### Notable Features Missing:
+
+* **App Blocking:** The functionality to block apps is not yet implemented but will be included in future updates.
+* **Cross-Platform Integration:** The desktop app currently does not integrate with our cross-platform system. This means it won't display usage data from your phone or browser extensions yet. Integration is planned for future releases.
+* **App Icons:** Custom app icons are not available yet. We will add these as we better understand the usage of this version of StayFree.
+* **Advanced Blocking Features:** Features like Focus Mode will be introduced in upcoming releases.
+* **Enhanced UI:** Future updates will bring more in-depth statistics, comparisons, and enhanced usability to the UI.
+* **Functionality Alignment:** Over time, the desktop app's functionality will closely match that of our browser extensions.
```

Comment:
Well, all of these are handled, so` Notable Features Missing` part can be removed completely.